src/en/comicextra/src/eu/kanade/tachiyomi/extension/en/comicextra/ComicExtra.kt

```diff
@@ -23,75 +23,75 @@ class ComicExtra : ParsedHttpSource() {
+                        }
+                    }
+                    is StatusFilter -> addQueryParameter("status", filter.selected)
                     else -> {}
                 }
             }
-            GET(url + if (page > 1) "/$page" else "", headers)
-        }
+        }.build()
+        return GET(url, headers)
     }
 
     override fun popularMangaFromElement(element: Element) = SManga.create().apply {
-        setUrlWithoutDomain(element.select("div.mb-right > h3 > a").attr("href"))
-        title = element.select("div.mb-right > h3 > a").text()
+        setUrlWithoutDomain(element.select("a.eg-image").attr("href"))
+        title = element.select("div.egb-right a").text()
         thumbnail_url = element.select("img").attr("src")
     }
 
     override fun latestUpdatesFromElement(element: Element) = SManga.create().apply {
-        setUrlWithoutDomain(element.select("div.hlb-t > a").attr("href"))
-        title = element.select("div.hlb-t > a").text()
-        thumbnail_url = fetchThumbnailURL(element.select("div.hlb-t > a").attr("href"))
+        setUrlWithoutDomain(element.select("a.big-link").attr("href"))
+        title = element.selectFirst(".big-link")!!.text()
     }
 
-    private fun fetchThumbnailURL(url: String) = client.newCall(GET(url, headers)).execute().asJsoup().select("div.movie-l-img > img").attr("src")
-
-    private fun fetchPagesFromNav(url: String) = client.newCall(GET(url, headers)).execute().asJsoup()
-
     override fun popularMangaNextPageSelector() = "div.general-nav > a:contains(Next)"
 
     override fun latestUpdatesNextPageSelector() = popularMangaNextPageSelector()
 
-    override fun searchMangaSelector() = popularMangaSelector()
+    override fun searchMangaSelector() = "div.dl-box"
 
-    override fun searchMangaFromElement(element: Element) = popularMangaFromElement(element)
+    override fun searchMangaFromElement(element: Element) = SManga.create().apply {
+        setUrlWithoutDomain(element.select("a.dlb-image").attr("href"))
+        title = element.select("div.dlb-right a.dlb-title").text()
+        thumbnail_url = element.select("a.dlb-image img").attr("src")
+    }
 
     override fun searchMangaNextPageSelector() = popularMangaNextPageSelector()
 
     override fun mangaDetailsParse(document: Document): SManga {
         return SManga.create().apply {
-            title = document.select("div.movie-detail span.title-1").text()
-            thumbnail_url = document.select("div.movie-l-img > img").attr("src")
-            status = parseStatus(document.select("dt:contains(Status:) + dd").text())
-            author = document.select("dt:contains(Author:) + dd").text()
-            description = document.select("div#film-content").text()
-            genre = document.select("dt.movie-dt:contains(Genres:) + dd a").joinToString { it.text() }
+            title = document.select("h1").text()
+            thumbnail_url = document.select("div.anime-image > img").attr("src")
```

Comment:
```suggestion
            thumbnail_url = document.select("div.anime-image > img").absUrl("src")
```

(this comment is independent of the other overlapping comment)